index.html

```diff
@@ -160,33 +171,35 @@ <h2>Multi-Post Stories</h2>
 
 
 <section class="aboutMe">
-    <h1>About<br>Myself</h1>
+    <section class="aboutInfo">
+        <h1>About<br>Myself</h1>
 
-    <p class="aboutPara">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="aboutPara">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
-    <h2 class="aboutLink">LET'S CONNECT</h2>
-        
-    <ul>
-        <li>
-            <a><img src="./img/gitHub.png"></a>
-            <a><img src="./img/linkedIn.png"></a>
-            <a><img src="./img/peace.png"></a>
-            <a><img src="./img/twitter.png"></a>
-            <a><img src="./img/m.png"></a>
-        </li>
-    </ul>
+        <h2 class="aboutLink">LET'S CONNECT</h2>
+            
+        <ul class="aboutSocial">
+            <li>
+                <a><img src="./img/gitHub.png"></a>
+                <a><img src="./img/linkedIn.png"></a>
+                <a><img src="./img/peace.png"></a>
+                <a><img src="./img/twitter.png"></a>
+                <a><img src="./img/m.png"></a>
+            </li>
+        </ul>
 
-    <button class="aboutMeButton">Get my resume</button>
+        <button class="aboutMeButton">Get My Resume</button>
+    </section>
 
     <section class="aboutMeDropdown">
         <ul>
             <li class="dropdownItemClick"><a class="dropdownTitleClick">Languages</a><img class="aboutArrowD" src="./img/arrowDwn.jpg"></li>
 
             <div class="ddList">
             <ul>
-                <li class="dropdownList"><img src="./img/jsIcon.png"><p>Javascript</p></li>
+                <li class="dropdownList"><img src="./img/jsIcon.png"><p>JavaScript</p></li>
                 <li class="dropdownList"><img src="./img/htmlIcon.png"><p>HTML</p></li>
-                <li class="dropdownList"><img src="./img/cssIcon.png"><p>CSS</p></li>
+                <li class="dropdownList"><img src="./img/cssIcon.png"><p>Css</p></li>
             </ul>
             </div>
```

Comment:
I have seen the following things that need to be redesigned according to the Desktop view.
These are two different sections according to the Figma Design. but in your design, I can see both sections on one page.
1. `font-size` of all elements
2. Alignment of all elements according to Desktop view
3. The width and height of About sections' text and Language part, and in Contact Section the form dimensions should have dimensions according to Desktop View.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221363819-a1afbf57-6058-4d75-9bb5-dc22ccb3cc55.png)| ![image](https://user-images.githubusercontent.com/61361037/221363732-2975543c-764e-4c59-9469-de7653043900.png)|